www/pages/plan-trip/plan-trip-controller.js

```diff
@@ -78,32 +72,31 @@ angular.module('pvta.controllers').controller('PlanTripController', function ($s
 
   //Called when this page is opened, and either a loaded trip has been queued
   //or there are no current existing parameters. Also called as a result of the
-  //newTrip method. Constructs the map, and then sets $scope.params as either default
+  //newTrip method. Constructs the map, and then sets $scope.request as either default
   //or loaded parameters
   var reload = function () {
     constructMap(defaultMapCenter);
     // All dates on this page are in Unix Epoch
-    currentDate = new Date();
-    $scope.params = {
-      name: 'New Trip',
-      time: {
-        datetime: currentDate,
-        type: 'departure',
-        asap: true
-      },
-      origin: {},
-      destination: {},
-      destinationOnly: true
-    };
+    $scope.noLocation = false;
```

Comment:
Oh yeah. Added that because we're reloading here, so we're resetting that flag that says 'We couldn't get your location'. If it was true before, we're setting it to false at the reload to give location services another chance. So we're not saying that we have the location, we're just saying that we haven't failed at getting it yet.